app/assets/stylesheets/blogs.scss

```diff
@@ -0,0 +1,37 @@
+.blogs {
+  .blog-header {
+    @include clearfix;
+
+    .icon {
+      float: left;
+      width: 64px;
+      height: 64px;
+    }
+
+    .title {
+      float: left;
+      margin-left: 20px;
+      line-height: 60px;
+    }
+  }
+
+  .blog-colophon {
+    margin-bottom: 10px;
+    font-size: 150%;
+    padding: 10px; // Same as $_content_padding
+    background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.